FutureNorthants/northants-website
# Frontend support for homepage hero logo image override

## Setup 

Relies upon these two PRs:
- https://github.com/FutureNorthants/northants-design-system/pull/106
- https://github.com/FutureNorthants/northants-website/pull/262

## Testing

- Checkout the design system branch, then run `yalc publish`
- Checkout the backend PR and import config
- Checkout this branch on the frontend and run `yalc add northants-design-system && yarn install && yarn dev`
- In a new window, run `yarn codegen` to ensure there are no changes to files
- Visit the Home page and expect no changes (unless you've already set an overridden logo)
- Edit the home page in the CMS to add a logo override and save/publish
- Refresh home page in frontend and expect to see the new logo
